walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,79 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Collections;
+import java.util.ArrayList;
+import java.util.Arrays;
 
 public final class FindMeetingQuery {
+  int FULL_DAY = 24*60;
+  Boolean[] freeStartTimes = new Boolean[FULL_DAY];
```

Comment:
Epoch has precision in seconds, and it is easy to achieve ms, nanoseconds, whatever someone wants. We cannot think of time as discrete.

Reply:
This also seems pretty inefficient to store the events - imagine if we have 1M such meetings, and all of them have their own freeStartTimes, but most of the fields would just be false (assuming most people don't have a lot of events).

How about storing all the events (start / end) time and loop through them to find availabilities?